plugins/mqtt/tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
     "compilerOptions": {
-        "module": "commonjs",
+        "module": "Node16",
```

Comment:
I dont think this is correct, you probably want this?

       "module": "CommonJS",
        "moduleResolution": "Node16",

Reply:
I will change if you like, but it usually helps me address odd Node issues during deployment. It is a valid module type. https://www.typescriptlang.org/tsconfig/#node16nodenext

Reply:
I will revert. It was likely troubleshooting that didn’t yield any change.

Reply:
I just noticed that my tsconfig is also complaining to change this, so I'm not sure what this is about. wasn't doing that before. might have been a node typings change. I'll investigate further later.